GHA macOS: "Certificate used to sign package is not trusted"

The GitHub Actions for macOS have started failing:

```
Install macOS dependencies
##[error]Process completed with exit code 1.
Run .github/workflows/macos-install.sh
installer: Package name is macOS_SDK_headers_for_macOS_10.14
installer: Certificate used to sign package is not trusted. Use -allowUntrusted to override.
##[error]Process completed with exit code 1.
```


Last pass, 2 days ago: https://github.com/python-pillow/Pillow/runs/271449358

First fail, 10 hours ago: https://github.com/python-pillow/Pillow/runs/274142645

Coincided with https://github.com/python-pillow/Pillow/issues/4167 on Windows.

--- a/.github/workflows/macos-install.sh
+++ b/.github/workflows/macos-install.sh
@@ -2,8 +2,6 @@
 
 set -e
 
-sudo installer -pkg /Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg -target /
-
 brew install libtiff libjpeg webp little-cms2
 
 PYTHONOPTIMIZE=0 pip install cffi

--- a/.travis/build.sh
+++ b/.travis/build.sh
@@ -3,5 +3,8 @@
 set -e
 
 coverage erase
+if [ $(uname) == "Darwin" ]; then
+    export CPPFLAGS="-I/usr/local/miniconda/include";
+fi
 make clean
 make install-coverage

